astroid/rebuilder.py

```diff
@@ -119,10 +120,16 @@ def _get_doc(self, node: T_Doc) -> Tuple[T_Doc, Optional[str]]:
                 ):
                     doc = first_value.value if PY38_PLUS else first_value.s
                     node.body = node.body[1:]
-                    return node, doc
+                    doc_node = nodes.Const(
+                        value=doc,
+                        lineno=first_value.lineno,
+                        col_offset=first_value.col_offset,
+                        parent=node,
```

Comment:
This should include the `end_line` and `end_column`. You can look at what is passed when we create other `Const` nodes here:
https://github.com/PyCQA/astroid/blob/6691a9e97373b28e15a76b747f78c041f2064df4/astroid/rebuilder.py#L1939-L1957